server/mdm/nanomdm/push/service/service.go

```diff
@@ -47,7 +47,7 @@ func New(store storage.PushStore, certStore storage.PushCertStore, providerFacto
 func (s *PushService) getProvider(ctx context.Context, topic string) (push.PushProvider, error) {
 	var (
 		err   error
-		stale bool = true
+		stale bool
```

Comment:
for a moment I was worried this might change things since the default value has changed, but seems like it's fine because it's reassigned before it's used at all, right?

Reply:
Exactly, the assignment was actually a cause of lint error because it was never used, it is assigned again before use.